src/app.js

```diff
@@ -208,6 +208,13 @@ export default function (app) {
       element.removeEventListener(event, oldValue)
       element.addEventListener(event, value)
 
+    } else if (name === "value" && "selectionEnd" in element && "selectionStart" in element) {
+      var selectionEnd = element.selectionEnd
```

Comment:
It's good to be verbose, but does the minifier pick this up and optimize it? Otherwise I'd prefer this to be inline.